WordPress/openverse
# Create a new `VIconButton`

## Fixes



Related to #460 by @fcoveram

## Description


This PR adds a new `VIconButton` component using the updated styles in the [Figma mockups](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?type=design&node-id=4658-12477&mode=design&t=3iXHyuTBcFAAWjLe-4)

The old styles for `VIconButton` were used mainly in the popover/modal close buttons, audio controls and the buttons within the search bar. They will gradually be updated to use the new `VIconButton`.

## Testing Instructions

Check out the new `VIconButton` story. It should have all of the styles from the Figma mockup. Old buttons (listed above) should also look the same.

Check the rendered Story: https://docs.openverse.org/_preview/2697/storybook/?path=/docs/components-viconbutton--default-story

## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
